docs/docusaurus/docs/core/expectation_suites/manage_expectation_suites.md

```diff
@@ -0,0 +1,294 @@
+
+  ```python name="core/expectation_suites/_examples/add_expectations_to_an_expectation_suite.py imports"
+  ```
+
+2. Get a Data Context.
+
+  ```python name="core/expectation_suites/_examples/add_expectations_to_an_expectation_suite.py get_context"
+  ```
+
+3. [Create a new](#create-a-new-expectation-suite) or [get an existing](#get-an-existing-expectation-suite) Expectation Suite.  This example retrieves an existing Expectation Suite.
+
+  ```python name="core/expectation_suites/_examples/add_expectations_to_an_expectation_suite.py get_suite"
+  ```
+
+4. Create an Expectation.
+
+  ```python name="core/expectation_suites/_examples/add_expectations_to_an_expectation_suite.py create an Expectation"
+  ```
+
+  The specific parameters provided when initializing an Expectation depend on the Expectation class.  You can view available Expectations and the parameters they take in the [Expectation Gallery](https://greatexpectations.io/expectations).
+
+
+5. Add the Expectation to the Expectation Suite.
+  ```python name="core/expectation_suites/_examples/add_expectations_to_an_expectation_suite.py add an Expectation to an Expectation Suite"
+  ```
+  :::tip 
+  
+  You can create an Expectation at the same time as you add it to the Expectation Suite:
+
+  ```python name="core/expectation_suites/_examples/add_expectations_to_an_expectation_suite.py create and add an Expectation"
+  ```
+  
+  :::
+
+<details><summary>Full example code</summary>
+<p>
+
+```python name="core/expectation_suites/_examples/add_expectations_to_an_expectation_suite.py full example code"
+```
+
+</p>
+</details>
+
+## Get an Expectation
+
+1. Import the GX Core library and `expectations` module.
+
+  ```python name="core/expectation_suites/_examples/get_a_specific_expectation_from_an_expectation_suite.py imports"
+  ```
+
+2. Get a Data Context.
+
+  ```python name="core/expectation_suites/_examples/get_a_specific_expectation_from_an_expectation_suite.py get_context"
+  ```
+
+3. [Get an existing Expectation Suite](#get-an-existing-expectation-suite) that contains Expectations, or [create a new Expectation Suite](#create-a-new-expectation-suite) and [add some Expectations to it](#add-expectations-to-an-expectation-suite).  This example retrieves an existing Expectation Suite.
+  ```python name="core/expectation_suites/_examples/get_a_specific_expectation_from_an_expectation_suite.py retrieve Expectation Suite"
+  ```
+
+4. Find the desired Expectation by iterating the Expectation Suite's Expectations and comparing classes and attributes to those of the desired Expectation.
+  ```python name="core/expectation_suites/_examples/get_a_specific_expectation_from_an_expectation_suite.py retrieve expectation"
+  ```
+
+<details><summary>Full example code</summary>
+<p>
+
+```python name="core/expectation_suites/_examples/get_a_specific_expectation_from_an_expectation_suite.py full example code"
+```
+
+</p>
+</details>
+
+## Edit a single Expectation
```

Comment:
```suggestion
## Edit a single Expectation
```
Remove. This is covered in Manage Expectations.

Reply:
I think that this should remain under manage Expectation Suites because the workflows they describe require an Expectation Suite -- eg, Edit a single Expectation _from an Expectation Suite._. This is a "thing you can do with an Expectation Suite" rather than strictly a "Thing you can do with an Expectation".